bedrock/products/templates/products/vpn/more/when-to-use.html

```diff
@@ -36,7 +36,7 @@ <h3>{{ ftl('vpn-when-to-use-5-in-your') }}</h3>
 <p>{{ ftl('vpn-when-to-use-because-your-internet') }}</p>
 
 <h3>{{ ftl('vpn-when-to-use-make-sure-you') }}</h3>
-<p>{{ ftl('vpn-when-to-use-a-vpn-provides', book=url('firefox.privacy.book')) }}</p>
+<p>{{ ftl('vpn-when-to-use-a-vpn-provides', book='https://support.mozilla.org/kb/how-does-phishing-and-malware-protection-work') }}</p>
```

Comment:
Hm. I'm not sure this is a good match for the context. 

> You still need to practice [good habits to stay safe online](https://support.mozilla.org/kb/how-does-phishing-and-malware-protection-work).

How about: https://support.mozilla.org/kb/how-stay-safe-web

Reply:
It definitely doesn't match the text, what was I thinking? Thanks!

```suggestion
<p>{{ ftl('vpn-when-to-use-a-vpn-provides', book='https://support.mozilla.org/kb/how-stay-safe-web') }}</p>
```